### PR TITLE
test: Run with web-test-runner instead of intern

### PR DESCRIPTION
**Pull request by Artur-**

### CODE REVIEW

File: flow-client/package.json [hunk 1]
```diff
@@ -26,33 +26,28 @@
...
+    "@koa/cors": "^4.0.0",
+    "@open-wc/dev-server-hmr": "^0.1.2-next.0",
+    "@open-wc/testing": "^3.1.7",
+    "@types/chai": "^4.3.4",
+    "@types/sinon": "^10.0.13",
     "@typescript-eslint/eslint-plugin": "^3.10.1",
-    "abort-controller": "2.0.0",
-    "chai": "4.2.0",
-    "esbuild-loader": "2.15.1",
+    "@web/dev-server": "^0.1.35",
```
Comment (sissbruecker) [inline flow-client/package.json:35]: `@koa/cors`, `@open-wc/dev-server-hmr` and `@web/dev-server` are not used.

File: flow-client/package.json [hunk 2]
```diff
@@ -26,33 +26,28 @@
...
-    "xhr-mock": "2.5.0"
+    "prettier": "^2.8.4",
+    "sinon": "^15.0.1",
+    "typescript": "^4.5.3"
   },
   "dependencies": {
     "@vaadin/common-frontend": "0.0.17",
-    "lit": "2.6.1"
+    "lit": "2.6.1",
+    "mock-xmlhttprequest": "^8.1.0"
```
Comment (sissbruecker) [inline flow-client/package.json:51]: Should this be a dev dependency?

File: flow-client/package.json (hunk 1 above)
Comment (Artur-) [reply flow-client/package.json]: Thanks, a bit too much copy paste from vaadin-dev-server

File: flow-client/package.json (hunk 2 above)
Comment (Artur-) [reply flow-client/package.json]: Yes, fixed

**Review by sissbruecker — APPROVED** [review]

**Pull request merged**